example/spec/helpers/granuleUtils.js

```diff
@@ -208,7 +208,18 @@ const waitForGranuleRecordsInList = async (stackName, granuleIds, additionalQuer
 
 const waitForGranuleRecordUpdatedInList = async (stackName, granule, additionalQueryParams = {}) => await pWaitFor(
   async () => {
-    const fieldsIgnored = ['timestamp', 'updatedAt'];
+    // Ignore the fields generated by CMR. The date string will be in a
+    // different ISO format.
+    const fieldsIgnored = [
+      'beginningDateTime',
+      'endingDateTime',
+      'error',
+      'files', // FUTURE: files contain a "type" field in elasticsearch that does not exist in Postgres
```

Comment:
We should add a comment to the ticket implementing this (it's mine at this point) to do this.    Adding so I don't forget.

Reply:
👍 https://bugs.earthdata.nasa.gov/browse/CUMULUS-2672?focusedCommentId=374019&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-374019

Reply:
Comment success!